charts/externalcreds/templates/deployment.yaml

```diff
@@ -176,6 +176,8 @@ spec:
         {{- end }}
         - name: SAM_ADDRESS
           value: {{ $.Values.samAddress }}
+        - name: SERVICE_GOOGLE_PROJECT
+          value: { { $.Values.serviceGoogleProject } }
```

Comment:
```suggestion
          value: {{ $.Values.serviceGoogleProject }}
```

Reply:
It wasn't actually getting through to the env, but was being defined in the helmfile - we just haven't needed it until now.